src/plugins/visualizations/public/vis_types/vis_type_alias_registry.ts

```diff
@@ -42,19 +42,15 @@ export interface VisualizationsAppExtension {
   }) => VisualizationListItem;
 }
 
-export interface VisTypeAliasPromotion {
-  description: string;
```

Comment:
As we don't have the section to promote a visualization with a description and a button I have removed this and changed this to a boolean in case we want in the future to visually differentiate the lens card